.github/workflows/release_maven.reusable.workflow.yml

```diff
@@ -0,0 +1,266 @@
+        required: true
+        default: 'pom.xml'
+        type: string
+      new-project-version:
+        description: |
+          (optional) The project version to be set in pom.xml and release.
+          Format : "5.7.0.2206241630"
+          When not specified, version is read from pom.xml and a timestamp is generated
+        required: false
+        type: string
+      maven-goals:
+        description: 'The goals for the mvn command (optional)'
+        required: false
+        type: string
+        default: 'clean deploy'
+      maven-other-options:
+        description: 'Maven options added at the end of the mvn command, overrides previous options'
+        required: false
+        type: string
+        default: '-DskipTests=false -DskipITs=false'
+      docker-file:
+        description: 'The path of the Dockerfile to be built. If empty, no docker build will occur'
+        required: false
+        type: string
+      symphony-microservice-name:
+        description: |
+          symphony-microservice-name is used to build and tag docker image.
+          Mandatory when docker-file is present
+        required: false
+        type: string
+        default: ''
+      docker-registry-project-name:
+        description: |
+          docker-registry-project-name is used to build and tag docker image.
+          Mandatory when docker-file is present
+        required: false
+        type: string
+        default: 'symphony-microservices'
+      kubernetes-update-deployment-descriptor:
+        description: 'Whether or not to update Kubernetes deployment descriptor so service is automatically deployed to cluster'
+        required: false
+        type: boolean
+        default: false
+      symphony-devops-workflows-ref:
+        description: 'symphony-devops-workflows reference branch (or tag) to be used. E.g. ''develop'''
+        required: false
+        type: string
+        default: 'develop'
+      runs-on-labels:
+        description: |
+          List of labels for the job runs-on clause (in JSON string array form,
+          can use either single or double quotes around values)
+          E.g. [ "ubuntu-latest" ] for using GitHub runners
+          E.g. [ "self-hosted", "linux" ] for using AVI-SPLI self-hosted runner
+        required: false
+        type: string
+        default: '[ "self-hosted", "linux" ]'
+    #
+    secrets:
+      # WARNING: secret names do not seem to accept "-" or "_" characters (not documented)
+      commitUserName:
+        description: User name to be used as committer
+        required: true
+      commitUserEmail:
+        description: User email to be used as committer
+        required: true
+      ciToken:
+        description: |
+          CI token is required for checking out and updating private repositories
+          GitHub token must be a PAT (Personal Access Token) which belongs to a user
+          who has read and write authorizations on the repositories
+        required: false
```

Comment:
if answer to previous question is yes, assuming this will be my PAT. But, wondering if this is not required, then will this manual workflow ask for my password?

Reply:
See new commit with updated documentation
This token belongs to the CI User, not to human user